circle.yml

```diff
@@ -4,9 +4,12 @@ machine:
   environment:
     BUILD_DIR: $HOME/$CIRCLE_PROJECT_REPONAME
     DB_NAME: "circle_test"
-    DB_USER: "ubuntu"
+    DB_USER: "root"
     DB_PASS: ""
     DB_HOST: "127.0.0.1"
+    DB_SOURCE_DUMP: "$BUILD_DIR/.data/db.dist.sql"
+    DB_CONTAINER_TAR: "$BUILD_DIR/.data/db-container.tar"
+    DB_CONTAINER_SCRIPT: "https://gist.githubusercontent.com/nicksantamaria/43f44c45ac2cb31298cacd5c546902ff/raw/a1092fe8602546944aca961e83b303f95c44a9e6/db-container-build.sh"
```

Comment:
can you please replace this variable with in-place value of `https://goo.gl/eoeTMb` - this redirects to an actual code, but I can track it and replace if required

Reply:
Yep, I've updated the script URL to `https://goo.gl/eoeTMb`